src/query/graphite/common/percentiles.go

```diff
@@ -71,6 +71,53 @@ func SafeSort(input []float64) int {
 	return nans
 }
 
+// SafeSum returns the sum of the input slice the number of NaNs in the input.
+func SafeSum(input []float64) (float64, int) {
+	nans := 0
+	sum := 0.0
+	for _, v := range input {
+		if !math.IsNaN(v) {
+			sum += v
+		} else {
+			nans += 1
+		}
+	}
+	return sum, nans
+}
+
+// SafeMax returns the maximum value of the input slice the number of NaNs in the input.
+func SafeMax(input []float64) (float64, int) {
+	nans := 0
+	max := -math.MaxFloat64
+	for _, v := range input {
+		if !math.IsNaN(v) {
+		if v > max {
+			max = v
+		}
```

Comment:
Indentation of inner of "if" contents is missing - needs further indenting. Looks like this file is not saved with "gofmt". Suprised linter hasn't caught this... Can you run "gofmt" on all these files?

Reply:
If you use a standard Go editor, it should run it for you on save. (i.e. vimgo, vscode w/ Go extensions, Go Land, etc)

Reply:
Yeah, I do edit with GoLand, which always catches stuff like this. The only reason this mis-indentation was included was because I manually did it on Github. A few days ago, I tried to make a trivial change, just so I could force the CI to re-run.